test/actions/store_test.exs

```diff
@@ -7,7 +7,13 @@ defmodule ArcTest.Actions.Store do
     use Arc.Actions.Store
     use Arc.Definition.Storage
 
-    def validate({file, _}), do: String.ends_with?(file.file_name, ".png")
+    def validate({file, _}) do
+      case String.ends_with?(file.file_name, ".png") do
```

Comment:
This is considered as a bad practice. `case` with only `true` and `false` should be rewritten as `if`.